cloudformation/aws-parallelcluster.cfn.json

```diff
@@ -2123,7 +2123,7 @@
                   "Ref": "ProxyServer"
                 },
                 "\n",
-                "custom_cookbook=",
+                "export custom_cookbook=",
```

Comment:
Don't do this but define the variable inside the env section for each command

Reply:
done 👍🏻